### PR TITLE
Wire instance create command to separated code

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

File: mcc/odin/cmd/instance_create.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/poka-yoke/spaceflight/mcc/odin/odin"
+)
+
+// instanceCreateCmd represents the instance create command
+var instanceCreateCmd = &cobra.Command{
+	Use:   "create [flags] identifier",
+	Short: "Creates a database",
+	Long:  `Creates a database, in RDS.`,
```
Comment (dsager) [inline mcc/odin/cmd/instance_create.go]: big difference between short & long :laughing:

**Comment by dsager** [comment]
+1

File: mcc/odin/cmd/instance_create.go (hunk 1 above)
Comment (ifosch) [reply mcc/odin/cmd/instance_create.go]: Yes! X)
This should be really redefined at some point.

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Comment by ifosch** [comment]
retest this

**Pull request closed without merging**